include/brod.hrl

```diff
@@ -17,104 +17,55 @@
 -ifndef(__BROD_HRL).
 -define(__BROD_HRL, true).
 
--type kafka_key()                 :: kpro:key().
```

Comment:
Understood! Will update the PR over the weekend :) 

Appreciate the late night feedback! 👍🏻